src/account-info/AccountDetails.js

```diff
@@ -66,19 +95,23 @@ export default function AccountDetails(props: Props): Node {
         />
         <ZulipText
           selectable
-          style={[styles.largerText, componentStyles.boldText, styles.halfMarginRight]}
+          style={[styles.largerText, componentStyles.boldText]}
```

Comment:
Hmm, puzzling that that was there.

It looks like the story here is that before 9badd24b90578b647476b496c8592b5f3a252191 / #4316, we had the presence icon at the right of the name, and this margin was a way of giving it some spacing from the name. With that commit, we moved the icon to the left of the name, but didn't notice this spacer margin and let it remain on the right.

Looking back at the before/after screenshots, one can see that there was less spacing "after" than "before":
https://github.com/zulip/zulip-mobile/pull/4316#issuecomment-734062265

The reduced spacing looks OK — and none of us have apparently noticed in the couple of years since we merged that — so it's fine to leave it out rather than putting it back.

Reply:
Oh, one commit-message nit:

    AccountDetails: Removed extra margin

Should be "Remove" rather than "Removed". Style guide:
https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#commit-messages